src/software/ai/hl/stp/action/pivot_action.cpp

```diff
@@ -36,7 +36,8 @@ void PivotAction::calculateNextIntent(IntentCoroutine::push_type& yield)
         {
             yield(std::make_unique<MoveIntent>(
                 robot->id(), pivot_point, (pivot_point - robot->position()).orientation(),
-                0.0, 0, enable_dribbler));
+                0.0, 0, enable_dribbler ? DribblerEnable::ON : DribblerEnable::OFF,
+                MoveType::NORMAL, AutokickType::NONE));
```

Comment:
Any particular reason we're not using `DribberEnable` in `MoveIntent` as well? I imagine it would be a bit cleaner if we could just pass it straight through, rather then having a inline ternary expression here?

Reply:
That's another large set of changes but might be worth doing in a separate ticket/PR?

Reply:
Not sure what ticket this is for (@EvanMorcom please link in description), but seems fairly in scope?

Reply:
This wasn't based on any ticket. I was just reviewing some stuff Saturday morning and decided to change MovePrimitive because it was so unclear.

I can make the change to MoveIntent as well

Reply:
@garethellis0 Just realized that this is Pivot not using DribblerEnable, not the Move Intent